### PR TITLE
Fixes: ESLint Configuration

### PR DESCRIPTION
**Pull request by ajomadlabs**
Moved the eslint config to package.json
Added jasmine for tests
Removed jest

Fixes #502

### CODE REVIEW

**Comment by ajomadlabs** [comment]
@matthewmorgan I have corrected it. Removed the nano.save.

File: package.json [hunk 1]
```diff
@@ -19,5 +19,21 @@
...
+    "env": {
+      "es6": true,
+      "node": true,
+      "jasmine": true 
+    },
+    "extends": "airbnb",
+    "rules": {
+      "import/no-unresolved": "off",
+      "import/extensions": "off"
+    }
```
[the remark this answers is not in the document]
Comment (tejasbubane) [reply package.json]: Just found out #456 does not mention about `.eslintrc.json` - it should have - sorry for the confusion. Just to clarify again - all you need to do is copy the config from `.eslintrc.json` to `package.json` - along with some json format/structure changes if required.

**Comment by ajomadlabs** [comment]
@tejasbubane Have made the required changes

**Comment by tejasbubane** [comment]
@ajomadlabs Can you remove the `.eslintrc.json` file? Since the config is now in package.json, we won't be needing that file now.

**Comment by ajomadlabs** [comment]
@tejasbubane I have removed the file. Just check if everything looks good

**Comment by tejasbubane** [comment]
Thanks @ajomadlabs!🎉 For future PRs please make changes and push to the same branch rather than closing and opening new PR. Looking forward to more contributions from you!

**Pull request merged**